tests/VectorException/host/host.c

```diff
@@ -13,6 +13,8 @@
 #define SKIP_RETURN_CODE 2
 
 static bool _was_ocall_called = false;
+extern const uint32_t supported_cpuid_leaves[OE_CPUID_LEAF_COUNT];
```

Comment:
Again redundant because VectorException.edl includes openenclave/internal/cpuid.h which already declares this.